style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  background-color: #3c3a39;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.landing {
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 15px;
+  background-color: #1c1a19;
+  color: white;
+  background-image: url("resources/Illustration.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
```

Comment:
- [ ] As it is a part of the design, please add the highlighted section in the image below to your own design and ensure you use the right spacing for all elements throughout your website.
![Screenshot 2023-08-01 at 09 31 45 (2)](https://github.com/Wisdomibu/Portfolio/assets/64774936/1530fd81-a966-4fb4-9454-b8d76abce204)